infra/src/main/kotlin/fi/fta/geoviite/api/frameconverter/v1/FrameConverterDataV1.kt

```diff
@@ -8,25 +8,27 @@ import fi.fta.geoviite.api.frameconverter.geojson.GeoJsonGeometry
 import fi.fta.geoviite.api.frameconverter.geojson.GeoJsonGeometryPoint
 import fi.fta.geoviite.api.frameconverter.geojson.GeoJsonProperties
 import fi.fta.geoviite.infra.common.AlignmentName
+import fi.fta.geoviite.infra.common.TrackMeter
 import fi.fta.geoviite.infra.common.TrackNumber
 import fi.fta.geoviite.infra.tracklayout.LocationTrackType
+import fi.fta.geoviite.infra.tracklayout.TrackLayoutTrackNumber
 import fi.fta.geoviite.infra.util.FreeText
 
 typealias FrameConverterIdentifierV1 = FreeText
 typealias FrameConverterResponseSettingsV1 = Set<FrameConverterResponseSettingV1>
 
 /**
- * @property BasicFeatureMatchData Response will include all fields from [BasicFeatureMatchDataV1]
- * @property FeatureGeometry Response will include geometry data, such as [GeoJsonGeometryPoint]
- * @property FeatureMatchDetails Response will include detailed data depending on request,
- * such as [CoordinateToTrackMeterConversionDetailsV1]
+ * @property FeatureMatchDataSimple Response will include all fields from [FeatureMatchDataSimpleV1]
+ * @property FeatureGeometryData Response will include geometry data, such as [GeoJsonGeometryPoint]
+ * @property FeatureMatchDataDetails Response will include detailed data depending on request,
+ *   such as [FeatureMatchDataDetailsV1]
```

Comment:
Nimittelyä vähän muuteltu, toivottavasti kuvaavammaksi. Kenttien nimien pitäisi myös nyt mätsätä paremmin näihin tyyppeihin sekaannusten välttämiseksi

Reply:
Data tapaa olla vähän redundantti sana olionimissä. Kaikkihan ne on dataa. Vähän niinkuin Type, Class tai Object - ei anna mitään hyödyllistä lisää nimen osana. Joskus toki silti käytössä - esim meillä silloin kun tekee kompositiolla ja rajapinnoilla joitain tyyppejä niin rajapinta + lopullinen tyyppi + välissä oleva kompositiotyyppi tarvitsee kaikki eri nimen niin noissa olen heittänyt sen Data-sanan erottamaan tuota kompositiotyyppiä.

Reply:
🤔 Mjooh ihan oikeassa olet. Tässä lienee päätynyt ajatus tuollaiseen "data on vain dataa", eikä kyseessä oleva dataluokka ole muu kontekstillinen asia, kuten kokonainen request/response mitä esim. muissa lähistöllä olevissa dataluokissa käytössä. Tässähän on myös hieman se pulma, että vastauksissa joissa näitä käytetään on vain osa "kokonaisdatasta" riippuen mitä pyynnössä haluttiin. Esim. Data-sana poistettua FeatureMatchSimple ei sinänsä tarkoita vielkään kokonaista asiaa, vaan se on potentiaalisesti vain osa sitä kokonaista palautettavaa datasettiä -> hieman hankalahko tulkita, muttei tuo Data-sana erityisesti liene tulkintaa auttavankaan.

Taidanpa nuo Data-sanat ottaa näistä siis veks